### DISCUSSION TITLE
[Feature] .dv file support (Digital Video Camera)

### DISCUSSION THREAD

**Discussion by CommanderRedYT** [immich-app/immich#12481] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I have some old camcorders which I would like to import into Immich. However, there is no support for `.dv` files at the moment.

ffmpeg has support for this so generating transcodes should not be a problem. However, it would be nice to import the actual files and not have to transcode it manually before being able to upload them into immich.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by mynameismaurizio** [comment]
Second this.

**Comment by MikNolhan** [comment]
I am in the same situation here, I have a whole archive, and would love to see them in there. At the least, and probably the best would be to include my MiniDV .dv files as some external library.
